### ISSUE TITLE
Link to ActivityPub follower list incorrect

### ISSUE THREAD

**Issue by nie-froehlich** [akirk/friends#390]
Installed are
- ActivityPub 4.2.0
- Friends 3.1.5

When I click on the number of followers in the "Friends Statistik" in the left hand menu, I get a list of followers (right now empty because it's a fresh install). On the page I have a link "View all followers in wp-admin" which points to `example.com/wp-admin/users.php?page=activitypub-followers-list`. 

When clicking that link, I get a "no access allowed to this page" error message.

Inside ActivityPub, when I want to view the list of followers, the link takes me to `example.com/wp-admin/options-general.php?page=activitypub&tab=followers`. 

Is this the link that Friends actually want to call? Or am I indeed missing an access permission to view the first page?

**Comment by akirk** [comment]
Thanks for pointing this out. I think the problem here is that in the ActivityPub plugin you have only activated the blog profile:
![Screenshot 2024-11-22 at 06 33 05](https://github.com/user-attachments/assets/cefb8e07-e9c7-4242-abf6-c18f2fe5b3fe)

Whereas this link points to the user profile, also the follower list comes from the user profile. I suppose the Friends plugin needs to account for the ActivityPub plugin mode here.

**Comment by nie-froehlich** [comment]
Yes, you're right, I currently have only the Blog profile active. Then I understand that the Followers section is so empty. I've enabled Author profiles and then the link worked.

As you said, it would be nice if Friends would take ActivityPub's mode into account, but it's not a very serious issue, I'd say.

**Comment by akirk** [comment]
I just released 3.1.7 in which there is now a Blog Followers item depending on your ActivityPub plugin settings.

**Comment by nie-froehlich** [comment]
getting closer: now the link to view all blog followers works, but I think the side bar is still a bit off. In mine it shows 0 blog followers whereas the Friends page itself shows the correct number of 2 followers.

![grafik](https://github.com/user-attachments/assets/88909457-746c-4b6c-ad91-ba71abec416a)

**Comment by akirk** [comment]
Oops, thanks! This was a typo, it will be fixed in the next release!
